docs/paper/admin/how-to/basic-troubleshooting.md

```diff
@@ -0,0 +1,135 @@
+---
+slug: /basic-troubleshooting
+---
+
+# Basic Troubleshooting
+
+This guide will help you diagnose your server's problem before reporting it to PaperMC or the plugin's author. Please, read this documentation carefully.
+
+:::tip Backup Your Server
+
+People can break their servers while trying to follow this guide. It is highly advised to backup your server before following this guide.  
+It would be ideal to create a test server by copying your production server's file, but that's not always possible.  
+Please, take a backup first.
+
+:::
+
+:::caution Stop Your Server
+
+Before following this guide, stop your server first. Modifying server files while it is still running will corrupt them. This is like having an operation without anesthesia. Your server will scream like you would.
+
+:::
+
+## Read The Error Message
+
+If your server encounters a problem, it will either print an error message on the server console, create a crash report and close itself, or do both.
+
+If your server crashes, the crash report will be saved in the `crash-report` directory.  
+If your server didn't crash, those error messages will be stored in the `log` directory along with other messages.
+
+The first thing you have to do is diagnose those messages.
+
+### Case 01: Paper Watchdog Thread Dump
+
+If your error message looks like this, **do not blindly report it to PaperMC** as it says:
+
+```plaintext
+[00:00:00] [Paper Watchdog Thread/ERROR]: --- DO NOT REPORT THIS TO PAPER - THIS IS NOT A BUG OR A CRASH - git-Paper-366 (MC: 1.19.3) ---
+[00:00:00] [Paper Watchdog Thread/ERROR]: The server has not responded for 10 seconds! Creating thread dump
+[00:00:00] [Paper Watchdog Thread/ERROR]: ------------------------------
+```
+
+This can be caused by various things. Maybe your server's hardware is not enough to run Minecraft server. Maybe one of the plugins you use is causing lag on your server.
+
+The thread dump from Paper Watchdog can be found below. If you find any plugin's name in there, talk to the plugin's author about that, not PaperMC.
+
+### Case 02: Stack Trace
+
+Almost every problem you encounter will print error message lines, which are called "stack trace", on the server console. Examining the stack trace will help you find out what is causing problems on your server.
```

Comment:
“… will print an error message, known as a “stack trace”, in the server console”

Reply:
I consider "error message" + "exception type" + "exception message" + "lines beginning with `at`" to be a single stack trace, so I wrote it that way. If I'm wrong, I will change that sentence.